packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -473,6 +474,10 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
   const hasPopupIcon = (!freeSolo || forcePopupIcon === true) && forcePopupIcon !== false;
 
   const { onMouseDown: handleInputMouseDown } = getInputProps();
+  const { ref: customlistboxRef } = ListboxProps ?? {};
+  const { ref: listboxRef } = getListboxProps();
```

Comment:
```suggestion
  const { ref: listboxRef, ...otherListboxProps } = getListboxProps();
```

then use `otherListboxProps` in line 674 below instead of calling the function again